frontend/src/static/js/components/webstatus-overview-filters.ts

```diff
@@ -192,6 +230,141 @@ export class WebstatusOverviewFilters extends LitElement {
+
+          const {browser, channel} = getBrowserAndChannel(browserColumnKey);
+          const browserLabel = BROWSER_ID_TO_LABEL[browser];
+          const channelLabel = CHANNEL_ID_TO_LABEL[channel];
+
+          if (channel === 'stable') {
+            columns.push(name);
+          }
+          columns.push(`${browserLabel} WPT ${channelLabel} Score`);
+        };
+
+        columnKeys.forEach(columnKey => {
+          const name = CELL_DEFS[columnKey].nameInDialog;
+          switch (columnKey) {
+            case ColumnKey.Name:
+              columns.push(name);
+              break;
+            case ColumnKey.BaselineStatus:
+              columns.push(name);
+              break;
+            case ColumnKey.StableChrome:
+            case ColumnKey.StableEdge:
+            case ColumnKey.StableFirefox:
+            case ColumnKey.StableSafari:
+            case ColumnKey.ExpChrome:
+            case ColumnKey.ExpEdge:
+            case ColumnKey.ExpFirefox:
+            case ColumnKey.ExpSafari:
+              pushBrowserChannelName(columnKey);
+              break;
+          }
+        });
+
+        // Convert array of feature rows into array of arrays of strings,
+        // in the same order as columns.
+        const rows = allFeatures.map(feature => {
+          const baselineStatus = feature.baseline?.status || '';
+          const browserImpl = feature.browser_implementations!;
+          const wptData = feature.wpt;
+          const row: string[] = [];
+
+          const pushBrowserChannelValue = (
+            browserColumnKey: BrowserChannelColumnKeys
+          ) => {
+            const {browser, channel} = getBrowserAndChannel(browserColumnKey);
+            const browserImplDate = browserImpl && browserImpl[browser]?.date;
+            const wptScore = wptData?.[channel]?.[browser]?.score;
+
+            if (channel === 'stable') {
+              row.push(browserImplDate || '');
+            }
+            row.push(String(wptScore) || '');
+          };
+
+          // Iterate over the current columns to get the values for each column.
+          for (const key of columnKeys) {
+            switch (key) {
+              case ColumnKey.Name:
+                row.push(feature.name);
+                break;
+              case ColumnKey.BaselineStatus:
+                row.push(baselineStatus);
+                break;
+              case ColumnKey.StableChrome:
+              case ColumnKey.StableEdge:
+              case ColumnKey.StableFirefox:
+              case ColumnKey.StableSafari:
+              case ColumnKey.ExpChrome:
+              case ColumnKey.ExpEdge:
+              case ColumnKey.ExpFirefox:
+              case ColumnKey.ExpSafari:
+                pushBrowserChannelValue(key);
+                break;
+            }
+          }
+          return row;
+        });
+
+        CSVUtils.downloadCSV(columns, rows, 'webstatus-feature-overview.csv')
```

Comment:
nit: we should probably move webstatus-feature-overview.csv to a constant (I know this wasn't the before but it would be nice now)

Reply:
done